docker/Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN apt-get update && \
  && rm -rf /var/lib/apt/lists/* /var/log/dpkg.log \
  && apt-get autoremove -y && apt-get clean -y
 
+RUN gem install cddl cbor-diag
+
```

Comment:
There is a gem install step further down that these could be added to, but see the meta-comment for more.